apps/emqx_modules/README.md

```diff
@@ -0,0 +1,54 @@
+# EMQX Modules
+
+The application provide some minor functional modules that are not included in the MQTT
+protocol standard, including "Delayed Publish", "Topic Rewrite", "Topic Metrics" and "Telemetry".
+
+
+## Delayed Publish
+
+After enabling this module, messages sent by the user with the prefix
+`$delayed/{Interval}/{Topic}` will be delayed by `{Interval}` seconds before
+being published to the `{Topic}`.
+
+More introduction about [Delayed Publish](https://www.emqx.io/docs/en/v5.0/mqtt/mqtt-delayed-publish.html).
+
+See [Enabling/Disabling Delayed Publish via HTTP API](https://www.emqx.io/docs/en/v5.0/admin/api-docs.html#tag/MQTT/paths/~1mqtt~1delayed/put).
+
+
+## Topic Rewrite
+
+Topic Rewrite allows users to configure rules to change the topic strings that
+the client requests to subscribe or publish.
+
+This feature is very useful when designing topics that are compatible with different
+client versions. For example, an old device that has already been issued and cannot
+be upgraded may use old topic rules, but the production environment need to apply
+a new design rules for the topics.
```

Comment:
This feature is useful to designing compatible topic system with different versions of client implements.
For example, a device will pub/sub message with topic `old/topic` with previous client system implement.
But the new devices implement doesn't contain the topic.
At this point, the topic-rewrite feature can be used to map the topics used by old devices to the topics used by the new system, achieving compatibility between new and old devices within the same system.